content/silver/concepts/automatic-attributes.md

```diff
@@ -12,6 +11,10 @@ propagate attr1, attr2, ...;
 ```
 This statement is overloaded for different kinds of attributes, forwarding to the appropriate equations on the production.
 
+# Inherited attributes
+If `env` is declared as an inherited attribute, then just writing `propagate env;` in a production body will generate copy equations for `env` on all children with the attribute.
+
+This is a recommended alternative to using [autocopy attributes](/silver/ref/decl/attributes/#autocopy-attributes), which may be deprecated at some point in the future.
```

Comment:
It might help to show an example like in all the other cases.  This one is easy but some readers may not know what a "copy equation" is.

For some reason I can't add a comment in the Monoid section below, so I'm writing it here. You give a fine description of monoid attributes (the first 2 sentences) but then there is a big block of code with no statement about what it is until **after** the code.  I think it would be better to say something about the problem - that it is computing `errors` and that the monoid here is `[]` and `++` and that the goal is simply to collect all the errors.  It is not as helpful to just show the code and then only after it have a sentence that says, effectively, you just read some code that does ABC, and it gets translated to the code below.

Reply:
The same pattern is repeated for functor attributes and that should be fixed too.